tests/js/sentry-test/utils.tsx

```diff
@@ -11,19 +12,12 @@ function findTextWithMarkup(contentNode: null | Element, textMatch: string | Reg
 }
 
 /**
- * Search for a text broken up by multiple html elements
+ * May be used with a *ByText RTL matcher to match text within multiple nodes
+ *
  * e.g.: <div>Hello <span>world</span></div>
  */
-export function getByTextContent(textMatch: string | RegExp) {
-  return screen.getByText((_, contentNode) => findTextWithMarkup(contentNode, textMatch));
-}
-
-/**
- * Search for *all* texts broken up by multiple html elements
- * e.g.: <div><div>Hello <span>world</span></div><div>Hello <span>world</span></div></div>
- */
-export function getAllByTextContent(textMatch: string | RegExp) {
-  return screen.getAllByText((_, contentNode) =>
-    findTextWithMarkup(contentNode, textMatch)
-  );
+export function textWithMarkupMatcher(textMatch: string | RegExp) {
```

Comment:
Instead of exporting these `get*` specific functions I'm just export a matcher factory so we can just pass it into any of the *ByText things.

cc @priscilawebdev

Reply:
Nice!!